include/cpu/mem.h

```diff
@@ -0,0 +1,84 @@
+// OsNES Project | include/mem.h
+// Description:
+// All NES memory objects are declared here.
+//
+// Authors: 
+// John Dengis
+//
+#ifndef MEM_H
+#define MEM_H
+#include "common.h"
+
+
+namespace Mem {
```

Comment:
This whole file is going to get junked later, so I'm not worrying about updating it.